src/server/game/Spells/SpellMgr.cpp

```diff
@@ -4650,6 +4650,153 @@ void SpellMgr::LoadSpellInfoCorrections()
 
     // ENDOF ANTORUS THE BURNING THRONE SPELLS
 
+    //
+    // SANCTUM OF DOMINATION
+    //
+
+    // Barbed Shot
+    ApplySpellFix({ 347807 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->AuraInterruptFlags |= SpellAuraInterruptFlags::LeavingCombat;
+        spellInfo->AuraInterruptFlags2 |= SpellAuraInterruptFlags2::EndOfEncounter;
+    });
+
+    // Ranger's Heartseeker
+    ApplySpellFix({ 352663 }, [](SpellInfo* spellInfo)
+    {
+        ApplySpellEffectFix(spellInfo, EFFECT_1, [](SpellEffectInfo* spellEffectInfo)
+        {
+            spellEffectInfo->ApplyAuraName = SPELL_AURA_PERIODIC_TRIGGER_SPELL;
+            spellEffectInfo->ApplyAuraPeriod = 180; // Sniffed value
+        });
+    });
+
+    // Domination Chain (Player)
+    ApplySpellFix({ 349451 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->AttributesEx2 |= SPELL_ATTR2_CAN_TARGET_NOT_IN_LOS;
+    });
+
+    // Wailing Arrow (Missile)
+    ApplySpellFix({ 348055 }, [](SpellInfo* spellInfo)
+    {
+        ApplySpellEffectFix(spellInfo, EFFECT_0, [](SpellEffectInfo* spellEffectInfo)
+        {
+            spellEffectInfo->BasePoints = 348056;
```

Comment:
why do you need this? this ID is set as TriggerSpell

Reply:
fixed on next commit.